test/dynamo/test_backends.py

```diff
@@ -106,6 +106,21 @@ def _check_backend_works(self, backend):
     def test_eager(self):
         self._check_backend_works("eager")
 
+    """
+    attribute lookup is not defined on builtin:
+    File "<eval_with_key>.8", line 8
+        child = self.getattr_L__self___layers___0___weight
+        child_1 = self.getattr_L__self___layers___0___bias
+        linear = torch.ops.aten.linear.default(l_x_, child, child_1);  l_x_ = child = child_1 = None
+                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
+        relu = torch.ops.aten.relu.default(linear);  linear = None
+        child_2 = self.getattr_L__self___layers___2___weight
+    """
+
+    @unittest.skipIf(
+        torch._dynamo.config.use_single_step_graph,
+        "attribute lookup is not defined on builtin",
+    )
     @_force_skip_lazy_graph_module()
     def test_torchscript(self):
         self._check_backend_works("ts")
```

Comment:
@ezyang This is also one of the real errors, it seems like ts backend can't handle the `torch.ops` in the fx graph. I am not sure what's the current status of the `ts` backend. I can find a way to always disable my thing for ts backend, or do you thing this is worth invesgating more?

Reply:
Nah just disable your thing for ts